### ISSUE TITLE
codes of your pretrained nets

### ISSUE THREAD

**Issue by dididadida** [kecsap/imagenet-clean#1]
Hi, Mr Kertész. You've done a great job and I'm very interested in your work, but I have problems in the process of reproduction. I use training codes of rwightman as you do and set the hyperparameters the same with yours. When I train enetb0 on the T, the results on the V and Vc are almost the same with yours. However, when I train enetb0 on the Tc, the top1 acc of enetb0 on the Vc is 81.19. 
I get the Tc by running these scripts: 
./imagenet_train_1_image_fixes.sh
./imagenet_train_2_image_removal.sh
./imagenet_train_3_categorical_fixes.sh
Is the way I get Tc wrong? Or are there other hyperparameters that I should set? And can you send me your codes of the squeezenet or shufflenet? The rwightman's codes do not contain these. So do you just import these from pytorch?
Thank you for your kind consideration of this request.

**Comment by kecsap** [comment]
Sorry for the late reply. After I apply the scripts to the training set and the validation to get Tc and Vc, I get similar accuracy to my paper after training: 83.24 %. The exact training command:

./distributed_train.sh 2 /DATASET/DIR/ --model efficientnet_b0 --sched cosine --lr 0.1 --reprob 0.5 --remode pixel --batch-size
110 --log-interval 2000 --epochs 90 --aa rand-m7-mstd0.5-inc1 --amp -j8

This is for two RTX 2070 GPUs. So the batch size, log interval might need different numbers depending on the available GPU memory and AMP is only meaningful for RTX GPUs.
